components/automate-ui/src/app/components/create-object-modal/create-object-modal.component.ts

```diff
@@ -20,10 +27,11 @@ export class CreateObjectModalComponent implements OnInit {
   @Output() close = new EventEmitter();
   @Output() createClicked = new EventEmitter<Project[]>();
 
+  public projects: { [id: string]: ProjectChecked } = {};
```

Comment:
Another spot to use the `ProjectMap` type, so this would become `public projects: projectMap = {};`